tasks/sql/medium/Select(select).sql

```diff
@@ -0,0 +1,14 @@
+-- # Which country has a population that is more than Canada but
+-- # less than Poland? Show the name and the population.
+-- # use select within select 
+
+-- sample table : 
+
+-- World (Name,Continent,Area,Population,GDP);
+
+-- |   Name       |  Continent  |     Area    |  Population  |     GDP     |
+-- |--------------|-------------|-------------|--------------|-------------|
+
```

Comment:
This would also need a real table so users could test if their solution works, but i think this is fine for now. Maybe they can just add the query and i will check if it looks right.